conda-envs/environment-dev.yml

```diff
@@ -5,8 +5,8 @@ channels:
 - defaults
 dependencies:
 # Base dependencies
-- aeppl=0.0.31
-- aesara=2.7.4
+- aeppl=0.0.32
```

Comment:
There's some other file, either pre-commit or mypy something that specifies aesara/aeppl versions that have to be updated as well.

Checking the last commit where these dependencies were changed should show it.

Reply:
Good catch! Last time we actually missed bumping Aesara in this location.

Reply:
We can just squash-merge once we get greens.

Reply:
No need, if you don't mind we squashing all the commits when merging. Otherwise you can.

We are less strict about this here than in Aesara for example

Reply:
Squash merge sounds great! :sweat_smile: